src/pages/home/sidebar/SidebarLinks.js

```diff
@@ -158,23 +158,35 @@ class SidebarLinks extends React.Component {
         this.state = {
             currentlyViewedReportID: props.currentlyViewedReportID,
             orderedReports: [],
+            priorityMode: props.priorityMode,
             unreadReports: SidebarLinks.getUnreadReports(props.reports || {}),
         };
     }
 
     static getDerivedStateFromProps(nextProps, prevState) {
         const shouldReorder = SidebarLinks.shouldReorder(nextProps, prevState.orderedReports, prevState.currentlyViewedReportID, prevState.unreadReports);
+        const switchingPriorityModes = nextProps.priorityMode !== prevState.priorityMode;
+
+        // Pull the reports we want to show on the left hand nav, ordered by priority
```

Comment:
This comment is a bit ambiguous. We are getting the formatted options to display from data that was already downloaded. Not sure how it was meant in this case.

Reply:
Maybe something like "Build the report options we want to show...."